recipes/ice-cream/build.sh

```diff
@@ -1,9 +1,16 @@
 #!/bin/bash
 set -eu -o pipefail
 
-mkdir -p ${PREFIX}/bin
-cp ICEcream.sh ${PREFIX}/bin/ICEcream.sh
-cp ICEfamily_refer/familytools/plotting_script.py ${PREFIX}/bin/plotting_script.py
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 
-chmod 0755 "${PREFIX}/bin/plotting_script.py"
-chmod 0755 "${PREFIX}/bin/ICEcream.sh"
\ No newline at end of file
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+
+cp -r * $outdir
```

Comment:
Can we here please be explicit? Otherwise, in new versions, we might copy way too much.

Reply:
Sure, I will revise accordingly. Thanks!